app/models/favorite.rb

```diff
@@ -0,0 +1,3 @@
+class Favorite < ApplicationRecord
+  belongs_to :breed
```

Comment:
This class has no other attributes besides its primary key ID and the breed foreign key.